packages/cadl-ranch-specs/http/azure/client-generator-core/access/mockapi.ts

```diff
@@ -58,15 +49,6 @@ Scenarios.Azure_ClientGenerator_Core_Access_RelativeModelInOperation = passOnSuc
       status: 200,
       body: json({ name: "Madge", inner: { name: "Madge" } }),
     },
-    handler: (req: MockRequest) => {
-      if (!("name" in req.query)) {
-        throw new ValidationError("Should submit name query", "any string", undefined);
-      }
```

Comment:
test failure here is due to difference of `request.params` and the validation.

validation here just check the existence of `name` in query.

Reply:
This can be updated to emitter test.